src/app/components/Byline/utilities/bylineExtractor/index.tsx

```diff
@@ -1,18 +1,17 @@
+import {
+  OptimoBylineContributorBlock,
+  OptimoBylineContributorMetadataBlock,
+} from '#app/models/types/optimo';
 import pathOr from 'ramda/src/pathOr';
```

Comment:
Did we manage to get rid of all references to `pathOr`?

Reply:
Theres still a reference in this file: https://github.com/bbc/simorgh/blob/6bb9d21540f7a645e89bf242624ba0dbc7e44693/src/app/components/Byline/utilities/bylineExtractor/index.tsx#L25

Its a tricky one to move off `pathOr`, as it constructs an array path to a nested block.model, like:
`pathOr('', ['some', 'long', 'path', here'], block)`.

Not sure of the best way of doing that outside of using `.reduce` perhaps?